scripts/evaluation.py

```diff
@@ -0,0 +1,288 @@
+import argparse
+import asyncio
+import os
+import signal
+import sys
+from asyncio import sleep
+from collections import defaultdict, deque
+from contextlib import asynccontextmanager
+from dataclasses import dataclass, field
+from pathlib import Path
+from time import perf_counter
+from typing import Any, AsyncGenerator, Deque, List, Optional, Tuple
+
+import numpy as np
+import pandas as pd
```

Comment:
Looks like an overkill for such simple script. Why not just a lists and dicts?

Look like this script should be runnable without and dependencies...